packages/portal-proto/src/features/GeneSummary/GeneSummary.tsx

```diff
@@ -280,6 +280,7 @@ const GeneView = ({
                   cohortFilters={cohortFilters}
                   genomicFilters={genomicFilters}
                   isModal={isModal}
+                  inModal={isModal}
```

Comment:
I feel like `isModal` is sufficient and we don't need a new prop`inModal`. Is there any reason why you added this when there is already `isModal`?

Reply:
from what i see `isModal` is for opining links in modals where `inModal` is for identifying if you are currently in a modal

Reply:
this is not the only place that the component is called, in other locations it has separate values